nservicebus/azure-storage-queues/multi-storageaccount-support.md

```diff
@@ -59,6 +61,28 @@ Each endpoint uses its own Azure storage account, thereby increasing message thr
 
 ![Scale out with multiple storage accounts](azure03.png "width=50%")
 
+## Securing connection strings to storage accounts for Scale Out
+
+[Securing connection strings to storage accounts](/nservicebus/azure-storage-queues/configuration.md#securing-connection-strings-to-storage-accounts) has been introduced in `NServiceBus.Azure.Transports.WindowsAzureStorageQueues` Version 7. When using a single account, securing connection string is limited to calling `.UseAccountNamesInsteadOfConnectionStrings()`. When Scaling Out is applied, securing connection strings requires using logical names of the accounts.
```

Comment:
> When using a single account, securing connection string is limited to calling `.UseAccountNamesInsteadOfConnectionStrings()`

Comparing this to [ASB](http://docs.particular.net/nservicebus/azure-service-bus/securing-connection-strings):

> To enable the same behavior for outgoing messages as well, use UseNamespaceNamesInsteadOfConnectionStrings() on the configuration API:

where "the same behavior" stands for:

> Versions 7 and above can map logical namespace name to a namespace connection string. If a valid mapping between a namespace name and a connection string doesn't exist a KeyNotFoundException is thrown.

This reads to me as two different outcomes of the similar feature. In ASB `UseAccountNamesInsteadOfConnectionStrings()` serves the purpose for outgoing messages. Where ASQ`s equivalent `UseAccountNamesInsteadOfConnectionStrings` is used to secure both incoming and outgoing messages for a single storage account case.

Thoughts?